### DISCUSSION TITLE
Student Github Copilot unavailable

### DISCUSSION THREAD

**Discussion by iprapas** [community/community#46269] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I used to be a verified student and for a month I didn't verify and lost the status. Now I am verified again but I can't get access to the free github copilot. I think this is a bug related to my temporary status change.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @iprapas 

Check if it is active on your account.

https://github.com/github-copilot/signup

**Reply by iprapas** [reply]
Only an option to sign up or start the trial

**Reply by SilasRodrigues19** [reply]
It should appear like this:

![image](https://user-images.githubusercontent.com/47532703/216619119-e528816a-0e6b-4860-9253-4e3e9ef7f421.png)

**Reply by iprapas** [reply]
You are right, that was it. Thank you
